support/client/lib/vwf/view/kineticjs.js

```diff
@@ -632,28 +619,26 @@ define( [ "module", "vwf/view", "jquery", "vwf/utility", "vwf/utility/color" ],
                     break;
 
                 case "visible":
-                    if ( pauseRendering ) {
+                    var visible = propertyValue;
+                    if ( this.state.pauseRendering ) {
                         drawThis = false;
-                    }
-                    else {
+                    } else {
                         drawThis = !activelyDrawing;
-                        if ( !propertyValue ) {
-                            clearBefore = !propertyValue;
-                        }
+                        clearBefore = visible ? clearBefore : false;
                     }
                     break;
 
                 case "pauseRendering":
+                    this.state.pauseRendering = propertyValue;
+                    drawThis = false;
```

Comment:
Yeah, I think it's correct as it is.  `drawThis` is the way for konva to know if it should redraw the object that just had a property change.  `scale` and `visible` both only want to redraw the object if the drawing is done, so they set it to `!activelyDrawing`.  However, we never want to redraw when the `pauseRendering` value changes (even when it changes to `true`) because it doesn't reflect a change to the visible state of the object.

I will admit that it seems like something is not 100% straightforward.  Maybe we should never draw if `pauseRendering` is `false`?  @landersk61 ... thoughts?

I'm going to go ahead and merge this as-is because it fixes the immediate bug while preserving the existing logic.  If we determine the logic needs some polishing, we can do that in a future PR.